llvm-hs/llvm-hs.cabal

```diff
@@ -61,7 +61,7 @@ custom-setup
 
 library
   build-tools: llvm-config
-  ghc-options: -Wall
+  ghc-options: -Wall -fno-warn-name-shadowing
```

Comment:
It is disabled in the llvm-hs-pure too for now to reduce the noise. I don't think it is a really significant warning. In the end we should enable it, but there are many other more problematic warnings to fix first.